src/Spout/Writer/AbstractWriter.php

```diff
@@ -69,17 +69,18 @@ public function setGlobalFunctionsHelper($globalFunctionsHelper)
      * By using this method, the data will be written to a file.
      *
      * @param  string $outputFilePath Path of the output file that will contain the data
+     * @param string $sheetName The custom name of the sheet
      * @return \Box\Spout\Writer\AbstractWriter
      * @throws \Box\Spout\Common\Exception\IOException If the writer cannot be opened or if the given path is not writable
      */
-    public function openToFile($outputFilePath)
+    public function openToFile($outputFilePath, $sheetName = null)
```

Comment:
This abstract class is meant to be generic and is also used for CSV. Since CSV does not support sheets, setting the name of the sheet should belong to the XLSX writer, and most likely to the sheet class, as it is already possible to work with sheets